### PR TITLE
Feature/multi assets

### PR DESCRIPTION
**Pull request by jplorek-atix**
This PR adds support for multiassets on the transaction and address pages
    
the assets are displayed on the following fields: 

- address balance tokens, 
- transaction minted tokens 
- transaction burned tokens 
- transaction outputs address 
- transaction input address

each field has:

- asset fingerprint
- -quantity

~~For this implementation the multi assets branch on the cardano graphql was used
https://github.com/input-output-hk/cardano-graphql/tree/feature/multi-asset~~

~~The `@cardano-graphql/client-ts` dependency version should be updated once the next `cardano-graphql` release containing multi-asset support is out~~

### CODE REVIEW

**Comment by cleverca22** [comment]
is `packages-cache/bignumber.js-9.0.1.tgz` supposed to be included in the git repo? i would expect it to be downloaded automatically

**Comment by lgobbi-atix** [comment]
> is `packages-cache/bignumber.js-9.0.1.tgz` supposed to be included in the git repo? i would expect it to be downloaded automatically

I think so, maybe @rhyslbw can clarify?

**Comment by rhyslbw** [comment]
> is packages-cache/bignumber.js-9.0.1.tgz supposed to be included in the git repo?

Yes, we use yarn offline package cache to facilitate reproducible builds.

File: source/features/address/ui/AddressSummary.tsx [hunk 1]
```diff
@@ -57,12 +75,32 @@ const AddressSummary = (props: AddressSummaryProps) => {
             <div className={styles.infoValue}>{props.transactionsCount}</div>
           </div>
           {isPaymentAddress(props) && (
-            <div className={styles.infoRow}>
-              <div className={styles.infoLabel}>
-                {translate('address.summaryBalanceLabel')}
+            <>
+              <div className={styles.infoRow}>
+                <div className={styles.infoLabel}>
+                  {/* TODO: add DE and JA translation*/}
```
Comment (rhyslbw) [inline source/features/address/ui/AddressSummary.tsx:81]: TODO: add DE and JA translation

File: source/features/address/ui/AddressSummary.tsx [hunk 2]
```diff
@@ -57,12 +75,32 @@ const AddressSummary = (props: AddressSummaryProps) => {
...
+                  {translate('address.adaBalance')}
+                </div>
+                <div className={styles.infoValue}>{props.finalBalance} ADA</div>
               </div>
-              <div className={styles.infoValue}>{props.finalBalance} ADA</div>
-            </div>
+              <div className={styles.infoRow}>
+                <div className={styles.infoLabel}>
+                  {/* TODO: add DE and JA translation*/}
```
Comment (rhyslbw) [inline source/features/address/ui/AddressSummary.tsx:88]: TODO: add DE and JA translation

File: source/features/transactions/components/TransactionInfo.tsx [hunk 3]
```diff
@@ -184,6 +194,40 @@ const TransactionInfo = (props: ITransactionInfoProps) => {
           </div>
         </div>
 
+        {/* ===== MINTED TOKENS ===== */}
+
+        {!isEmpty(props.mint) && (
+          <div className={classnames([styles.idRow, styles.row])}>
+            <div className={styles.label}>
+              {/* TODO: add DE and JA translation*/}
```
Comment (rhyslbw) [inline source/features/transactions/components/TransactionInfo.tsx:202]: TODO: add DE and JA translation

File: source/features/transactions/components/TransactionInfo.tsx [hunk 4]
```diff
@@ -184,6 +194,40 @@ const TransactionInfo = (props: ITransactionInfoProps) => {
...
+            </div>
+          </div>
+        )}
+
+        {/* ===== BURNED TOKENS ===== */}
+
+        {!isEmpty(props.burn) && (
+          <div className={classnames([styles.idRow, styles.row])}>
+            <div className={styles.label}>
+              {/* TODO: add DE and JA translation*/}
```
Comment (rhyslbw) [inline source/features/transactions/components/TransactionInfo.tsx:219]: TODO: add DE and JA translation

**Review by rhyslbw — CHANGES_REQUESTED** [review]
Great work. We just need DE and JA translations for the following phrases:

- Ada Balance
- Tokens balance
- Policy ID
- This address has no tokens
- Minted tokens
- Burned tokens

I'll sort this out.

**Comment by eddyjlhaigh** [comment]
Here are the Japanese translations:

Ada Balance ADA残高
Tokens balance トークン残高
Policy ID ポリシーID
This address has no tokens このアドレスにはトークンがありません
Minted tokens 鋳造済みトークン
Burned tokens バーン済みトークン

**Review by eddyjlhaigh — CHANGES_REQUESTED** [review]
Looks good accept for one minor issue. The "Tokens balance" is not horizontally aligned with an addresses tokens, as seen here https://explorer.mary-qa.dev.cardano.org/en/address.html?address=addr_test1qz3gkzmatgtvw7zdkttvchftja4fys79krn7rue6x7dc43t939ww4vt849h8wpjv38czxt4q97vc8438tte28amvwm3qm63e2w. I played with the element and found changing the `margin-top` from `5px` to `0px` on the `TransactionTokenList_token` fixed this.

**Comment by DominikGuzei** [comment]
Hey @rhyslbw I have pushed the DE translations … not sure if it's relevant but I saw this error when running `yarn dev`:
grafik

**Comment by DominikGuzei** [comment]
hm … really strange, my GIT client says everything is up-to-date (pushed) but here on Github my commit with the German translations does not seem to appear 🤔

**Comment by rhyslbw** [comment]
Thanks @DominikGuzei 
Ah sorry I should have warned you about the dev backend needed to be rebuilt. The error is just reflecting it's not in sync yet

**Pull request merged**